src/current/v23.2/export.md

```diff
@@ -60,19 +58,25 @@ No special privilege is required for:
 
 ## Parameters
 
- Parameter | Description
+Parameter | Description
 -----------|-------------
- `file_location` | Specify the [URL of the file location](#export-file-url) where you want to store the exported data.<br><br>Note: It is best practice to use a unique destination for each export, to avoid mixing files from different exports in one directory.
- `opt_with_options` | Control your export's behavior with [these options](#export-options).
- `select_stmt` | Specify the query whose result you want to export.
- `table_name` | Specify the name of the table you want to export.
+`file_location` | Specify the [URL of the file location](#export-file-url) where you want to store the exported data.<br><br>Note: It is best practice to use a unique destination for each export, to avoid mixing files from different exports in one directory.
+`opt_with_options` | Control your export's behavior with [these options](#export-options).
+`select_stmt` | Specify the query whose result you want to export.
+`table_name` | Specify the name of the table you want to export.
 
 ### Export file URL
 
 You can specify the base directory where you want to store the exported files. CockroachDB will create the export file(s) in the specified directory with programmatically generated names (e.g., `exportabc123-n1.1.csv`, `exportabc123-n1.2.csv`, `exportabc123-n2.1.csv`, ...). Each export should use a unique destination directory to avoid collision with other exports.
 
 The `EXPORT` command [returns](#success-responses) the list of files to which the data was exported. You may wish to record these for use in subsequent imports.
 
+If an export encounters some kind of failure or [cancelation](#cancel-a-running-export), it will leave any written files behind in the destination. To run a new export and avoid collision with previously written export files, consider doing the following:
```

Comment:
Please correct the spelling of "cancelation" to "cancellation".

Reply:
thanks Florence! I find it easy to forget when US English doubles and doesn't double the L. Both are accepted, but the double L is preferred. Good to remember that!